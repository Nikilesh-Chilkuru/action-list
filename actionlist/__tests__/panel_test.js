/* eslint-env node, jest */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Panel from '../panel';

test('renders correctly', () => {
  const tree = renderer.create(
    <Panel />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
