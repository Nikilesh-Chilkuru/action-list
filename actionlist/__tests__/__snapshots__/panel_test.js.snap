exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#e9e9e9",
      "borderBottomWidth": 0.5,
      "overflow": "hidden",
      "paddingRight": 10,
    }
  }>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#7B1500",
          "flex": 1,
          "fontWeight": "bold",
          "padding": 10,
        }
      } />
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          undefined,
        ]
      }
      testID={undefined}>
      <Image
        source={1}
        style={
          Object {
            "height": 20,
            "marginTop": 10,
            "width": 20,
          }
        } />
    </View>
  </View>
</View>
`;
